### ISSUE TITLE
Remove all banners for our features

### ISSUE THREAD

**Issue by ryankscott** [sourcegraph/sourcegraph#38625]

**Comment by taras-yemets** [comment]
@ryankscott, 

<details>
<summary>Banners we're going to remove</summary>
<img width="1712" alt="Screenshot 2022-07-13 at 10 05 05" src="https://user-images.githubusercontent.com/25318659/178677414-774b0d8a-571f-4110-8692-109a55d2d1b5.png">
<img width="1712" alt="Screenshot 2022-07-13 at 09 58 27" src="https://user-images.githubusercontent.com/25318659/178677427-daa5276a-0290-4416-a5cd-4a6c01de4699.png">
<img width="1712" alt="Screenshot 2022-07-13 at 10 12 32" src="https://user-images.githubusercontent.com/25318659/178677531-babec0f5-e11a-447e-8453-208e46e4cad0.png">
</details>

By the way, do we want to change the signup banner copy (regarding private repos)?
<img width="1712" alt="Screenshot 2022-07-13 at 10 03 00" src="https://user-images.githubusercontent.com/25318659/178677787-4d5fbbe0-6055-4134-84de-00dde8f94741.png">

**Comment by ryankscott** [comment]
Yeah I think we probably want to drop that one too. I'll leave that up to @ryphil to decide.

**Comment by ryankscott** [comment]
@taras-yemets I just talked to @ryphil we should remove that signup banner too.
